generate_helm_values.py

```diff
@@ -0,0 +1,285 @@
+# Copyright (C) 2024 Intel Corporation
```

Comment:
I would suggest we don't introduce this separate file and merge it into deploy.py. 

This is to keep GenAIExamples root dir clean enough

Reply:
Integrated it with deploy.py